docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Getting started
+order: 1
+slug: getting-started-future
+lang: en
+layout: tutorial
+permalink: /tutorial/getting-started-future
+---
+
+## Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will cover how to get started with your next-gen platform using Bolt for Python, by setting up the Slack CLI and installing the required dependencies.
+
+Find out about the next-generation platform on Slack's <a href="https://api.slack.com/future/intro" target="_blank">official introduction page.</a>
+</div>
+
+---
+
+### Setting up {#setting-up}
+
+#### Slack CLI {#setting-up-cli}
+
+To build a next-generation app with Bolt for Python, you'll need to get the Slack CLI.
+
+Install the Slack Cli by following this <a href="https://api.slack.com/future/quickstart" target="_blank">Quickstart</a>. Since we won't be using Deno to build our next-generation app, you can skip any instructions related to installing Deno or creating an app using a Deno template. Once you've logged into the CLI using `slack login` and verified your login using `slack auth list`, you can proceed with the instructions in this guide.
```

Comment:
```suggestion
Install the Slack CLI by following this <a href="https://api.slack.com/future/quickstart" target="_blank">Quickstart</a>. Since we won't be using Deno to build our next-generation app, you can skip any instructions related to installing Deno or creating an app using a Deno template. Once you've logged into the CLI using `slack login` and verified your login using `slack auth list`, you can proceed with the instructions in this guide.
```